Tasks/MavenV3/mavenutil.ts

```diff
@@ -193,64 +188,78 @@ interface RepositoryInfo {
     id:string;
 }
 
-function collectFeedRepositories(pomContents:string): Q.Promise<any> {
-    return convertXmlStringToJson(pomContents).then(function (pomJson) {
+async function collectFeedRepositories(pomContents:string): Promise<any> {
+    return convertXmlStringToJson(pomContents).then(async function (pomJson) {
```

Comment:
Seems we are mixing async and Promise style together.  Maybe cleaner to just await for the `convertXmlStringToJson`, and write it as imperative style.